### DISCUSSION TITLE
time schedule missing

### DISCUSSION THREAD

**Discussion by everdrift** [marcokreeft87/formulaone-card#283] [Q&A]
Am I the only one mising the time schedule in the "next_race"card like practice, quali and race time/dates?
![Next_race](https://github.com/marcokreeft87/formulaone-card/assets/83213448/0b92e20a-5cac-41b1-a65f-326bfcabb3be)

Those were there in the past and in the examples
(v 1.8.6)

![next_race_01](https://github.com/marcokreeft87/formulaone-card/assets/83213448/d3bc964f-80e0-4c58-a6c8-efee0b62ad89)

**Comment by marcokreeft87** [comment]
Try show_raceinfo: true

**Reply by everdrift** [reply]
Thanks, that did the trick! Happy family here…..
